test/Model/MongoConnectedListenerTest.php

```
@@ -61,8 +65,8 @@ public function testCreate()
     /**
      * @param mixed $lastId
      * @return mixed
-     * @depends testCreate
      */
+    #[Depends('testCreate')]
```

PHPStan (2.2.7)

[error] PHPDoc tag `@param` for parameter $lastId with type mixed is not subtype of native type string.


(parameter.phpDocType)